### ISSUE TITLE
Window resize not works with addon

### ISSUE THREAD

**Issue by strayge** [drcika/apc-extension#9]
vscode stopped properly resize after update addon to 0.0.6

vscode version: 1.79.0-insider

Possibly related issues:
- vscode about window not showing
- "toggle developer tools" also do nothing

"Disable Apc extension" fixes everything.

### Initial state after load

![image](https://github.com/drcika/apc-extension/assets/2664578/eba08fc3-c684-4211-86fc-92b0e76fb271)

### Resize vscode window

![image](https://github.com/drcika/apc-extension/assets/2664578/1bfcab21-0329-49d2-ba89-31de4ede22e7)

### Change apc settings and save them to fix

next resize also broke vscode

![image](https://github.com/drcika/apc-extension/assets/2664578/0acd9641-99eb-49e6-8d2c-0e3ea52ff959)

**Comment by strayge** [comment]
fixed by commenting [this](https://github.com/drcika/apc-extension/blob/43aa902f12b3cc1a8308544117843b1059b33c9d/modules/patch.js#L157) line

at this time `this.globalActivitiesContainer` is `false`
so another fix for me (at few lines above) is change:
```js
this.globalActivitiesContainer = content?.childNodes instanceof Array && content.childNodes[1];
```
to
```js
this.globalActivitiesContainer = content?.childNodes && content.childNodes[1];
```

**Comment by strayge** [comment]
fixed in 0.0.8
